### ISSUE TITLE
`no-physical-properties` rule fails to detect `textAlign: "left"` as physical properties

### ISSUE THREAD

**Issue by azu** [chakra-ui/eslint-plugin-panda#242]
The `@pandacss/no-physical-properties` ESLint rule is designed to detect physical CSS properties and suggest logical properties instead.

- https://github.com/chakra-ui/eslint-plugin-panda/blob/main/docs/rules/no-physical-properties.md

However, it currently fails to detect `textAlign: "left"` and `textAlign: "right"` as physical properties that should be replaced with logical properties.

## Reproduction

I've created a reproduction repository at: https://github.com/azu/no-physical-properties-text-align

Steps to reproduce:

1. Clone the repository
2. Install dependencies with `pnpm install`
3. Run ESLint with `pnpm run lint`

The ESLint rule does not report any errors for the following code in `src/app/page.tsx`:

```tsx
import { css } from "../../styled-system/css";
export default function Home() {
  return (
    <main>
      <p className={css({
        textAlign: "left" // This should trigger an ESLint error
      })}>left text</p>
      <p className={css({
        textAlign: "right" // This should trigger an ESLint error
      })}>right text</p>
      <p className={css({
        textAlign: "start" // This is valid
      })}>start text</p>
      <p className={css({
        textAlign: "end"  // This is valid
      })}>end text</p>
    </main>
  );
}
```

## Expected Behavior

The ESLint rule should detect `textAlign: "left"` and `textAlign: "right"` as physical properties and suggest using the logical properties `textAlign: "start"` and `textAlign: "end"` instead.

According to [MDN Documentation](https://developer.mozilla.org/en-US/docs/Web/CSS/text-align), using logical properties like `start` and `end` is recommended for better internationalization support, especially for RTL languages.

## Config

https://github.com/azu/no-physical-properties-text-align/blob/main/.eslintrc.json

```json
{
  "plugins": [
    "@pandacss"
  ],
  "extends": [
    "next/core-web-vitals",
    "plugin:@pandacss/recommended"
  ],
  "rules": {
    "@pandacss/no-physical-properties": "error"
  }
}
```
